docs/Introduction/architecture-overview/architecture-request-model.md

```diff
@@ -31,7 +31,7 @@ Learn more about [ERC-677 and the LINK token](../link-token-contracts/).
 
 ### Oracle Contract
 
-[`Oracle` contracts](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/Oracle.sol) are owned by oracle node operators, which run alongside off-chain oracle nodes.
+[`Oracle` contracts](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.8/interfaces/OracleInterface.sol) are owned by oracle node operators, which run alongside off-chain oracle nodes.
```

Comment:
```suggestion
[`Oracle` contracts](https://github.com/smartcontractkit/chainlink/blob/master/contracts/src/v0.6/Oracle.sol) are owned by oracle node operators, which run alongside off-chain oracle nodes.
```

Reply:
Hello @yannluppo 

thanks so much for raising this issue. See my suggestion, I prefer to point readers to the implementation of the Oracle contract since it provides more information

Reply:
Applied